src/routes/Metatag.svelte

```diff
@@ -0,0 +1,41 @@
+<script>
+    import { MetaTags } from 'svelte-meta-tags';
+</script>
+
+<MetaTags 
+		title="Home page" 
+		titleTemplate="SCIS | %s" 
+		description="scis web" 
+		canonical="https://localhost:5173/"
```

Comment:
Use env vars instead of hard coding

Reply:
for title , description as well ? and do i have to use "import.meta.env.VARIABLE_NAME" to access the env ?